src/middleware/externalRequests.js

```diff
@@ -364,7 +366,7 @@ const sendMappedObject = (
           reqTimestamp,
           responseTimestamp,
           request.id,
-          error,
+          orchestrationError,
```

Comment:
```suggestion
          error: orchestrationError,
```

Reply:
@MattyJ007 why do we have to do this? it causes things to break as well

Reply:
Sorry, that's my bad - I misread what was happening - thought you were passing one object to `createOrchestration` - didn't realise those were parameters. Thought `createOrchestration` was extracting the `error` field from the incoming object...
Ignore my nonsense. Previous one as well :|

Reply:
okay cool